### PR TITLE
chore: deprecate buildx args

### PR DESCRIPTION
**Pull request by aarnphm**
This PR revert `containerize` to use `docker build` with BuildKit.

`buildx` is currently not yet supported on some CI vendor.

Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

```bash
$ bentoml containerize --help
Usage: bentoml containerize [OPTIONS] BENTO_TAG

  Containerizes given Bento into a ready-to-use Docker image.

  BENTO is the target BentoService to be containerized, referenced by its name
  and version in format of name:version. For example: "iris_classifier:v1.2.0"

  'bentoml containerize' command also supports the use of the 'latest' tag which will automatically use the last built
  version of your Bento.

  You can provide a tag for the image built by Bento using the '--docker-image-tag' flag. Additionally, you can
  provide a '--push' flag, which will push the built image to the Docker repository specified by the image tag.

  You can also prefixing the tag with a hostname for the repository you wish to push to. e.g. 'bentoml containerize
  IrisClassifier:latest --push --tag repo-address.com:username/iris' would build a Docker image called
  'username/iris:latest' and push that to docker repository at repo-address.com.

  By default, the 'containerize' command will use the current credentials provided by Docker daemon.

  'bentoml containerize' also uses Docker Buildx as backend, in place for normal 'docker build'. By doing so, BentoML
  will leverage Docker Buildx features such as multi-node builds for cross-platform images, Full BuildKit capabilities
  with all of the familiar UI from 'docker build'.

Options:
  -t, --docker-image-tag TEXT    Name and optionally a tag (format: 'name:tag'), defaults to bento tag.
  --add-host TEXT                Add a custom host-to-IP mapping (format: 'host:ip').
  --build-arg TEXT               Set build-time variables.
  --cache-from TEXT              External cache sources (e.g., 'user/app:cache', 'type=local,src=path/to/dir').
  --iidfile TEXT                 Write the image ID to the file.
  --label TEXT                   Set metadata for an image.
  --network TEXT                 Set the networking mode for the 'RUN' instructions during build (default 'default').
  --no-cache                     Do not use cache when building the image.
  --output TEXT                  Output destination (format: 'type=local,dest=path').
  --platform TEXT                Set target platform for build.
  --progress [auto|tty|plain]    Set type of progress output ('auto', 'plain', 'tty'). Use plain to show container
                                 output.
  --pull                         Always attempt to pull all referenced images.
  --secret TEXT                  Secret to expose to the build (format: 'id=mysecret[,src=/local/secret]').
  --ssh TEXT                     SSH agent socket or keys to expose to the build (format:
                                 'default|<id>[=<socket>|<key>[,<key>]]').
  --target TEXT                  Set the target build stage to build.
  --enable-features [features,]  Enable additional BentoML features. Available features are: tracing, grpc, grpc-
                                 reflection, grpc-channelz, aws, all, io-image, io-pandas, io-json, tracing-zipkin,
                                 tracing-jaeger, tracing-otlp.
  --allow TEXT                   DEPRECATED (buildx argument): Allow extra privileged entitlement (e.g.,
                                 'network.host', 'security.insecure'). (no-op, will be removed)
  --build-context TEXT           DEPRECATED (buildx argument): Additional build contexts (e.g., name=path). (no-op,
                                 will be removed)
  --builder TEXT                 DEPRECATED (buildx argument): Override the configured builder instance. (no-op, will
                                 be removed)
  --cache-to TEXT                DEPRECATED (buildx argument): Cache export destinations (e.g., 'user/app:cache',
                                 'type=local,dest=path/to/dir'). (no-op, will be removed)
  --cgroup-parent TEXT           DEPRECATED (buildx argument): Optional parent cgroup for the container. (no-op, will
                                 be removed)
  --metadata-file TEXT           DEPRECATED (buildx argument): Write build result metadata to the file. (no-op, will
                                 be removed)
  --ulimit TEXT                  DEPRECATED (buildx argument): Ulimit options (default []). (no-op, will be removed)
  --no-cache-filter TEXT         DEPRECATED (buildx argument): Do not cache specified stages. (no-op, will be removed)
  --shm-size TEXT                DEPRECATED (buildx argument): Size of '/dev/shm'. (no-op, will be removed)
  --load TEXT                    DEPRECATED (buildx argument): Shorthand for '--output=type=docker'. Note that '--
                                 push' and '--load' are mutually exclusive. (no-op, will be removed)
  --push TEXT                    DEPRECATED (buildx argument): Shorthand for '--output=type=registry'. Note that '--
                                 push' and '--load' are mutually exclusive. (no-op, will be removed)
  -q, --quiet                    Suppress all warnings and info logs
  --verbose, --debug             Generate debug information
  --do-not-track                 Do not send usage info
  -h, --help                     Show this message and exit.


```

### CODE REVIEW

File: examples/quickstart/bentofile.yaml [hunk 1]
```diff
@@ -8,5 +8,3 @@ python:
   packages:
     - scikit-learn
     - pandas
-docker:
-  env: ./.env
```
Comment (sauyon) [inline examples/quickstart/bentofile.yaml:12]: Let's put this into another PR.

Comment (aarnphm) [reply examples/quickstart/bentofile.yaml:12]: done

**Comment by aarnphm** [comment]
In favor of #3164
